js/dashboard.js

```diff
@@ -199,6 +199,110 @@ $(function() {
         }
     });
 
+    $('#leafwetness-button').click(function() {
+				$.ajax({
+            url: 'http://localhost:8000/env_data?vineyard_id=0&env_variable=leafwetness',
+            type: "GET"
+        }).done(function(json) {
+            if (json.env_data) {
+                window.location.href = "http://localhost:8000/env_data?vineyard_id=0&env_variable=leafwetness";
+            } else {
+                // go nowhere
```

Comment:
But we're not including that for MVP, right? I don't think placeholders are necessary right now. And if we do leave them in there, we should add a TODO line.